packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx

```diff
@@ -71,8 +76,8 @@ const createStyleSheet = (brandTheme: Brand, theme: Theme): StyleSheetType =>
     button: {
       height: 52,
       borderRadius: 7,
-      marginBottom: theme.spacing.large,
-      marginTop: theme.spacing.large,
+      marginBottom: 32,
```

Comment:
are we dropping the usage of theme ?

Reply:
https://coorpacademy.slack.com/archives/GA45KH3UM/p1674553275788439?thread_ts=1674488454.388169&cid=GA45KH3UM